schemas/glean/glean/glean.1.schema.json

```diff
@@ -770,7 +770,7 @@
         },
         "ping_type": {
           "maxLength": 30,
-          "pattern": "^[a-z_][a-z0-9_]*$",
+          "pattern": "^[a-z-][a-z0-9-]*$",
```

Comment:
Don't we still need to support the old underscore-style pings?

Reply:
Good point, @badboy.  We need to support both indefinitely (at least at this level).